packages/lodestar/src/chain/chain.ts

```diff
@@ -59,7 +59,7 @@ export class BeaconChain extends (EventEmitter as { new(): ChainEventEmitter })
   public chainId: Uint16;
   public networkId: Uint64;
   public clock: IBeaconClock;
-  private epochCtx: EpochContext;
```

Comment:
I wonder if it would be better to expose getter to copy of EpochContext as this posses danger that some code might change EpochContext accidentally. 

We could expose getBeaconProposers and getBeaconCommittee from chain which we also need in api.

Reply:
:+1: definitely anything returned by the API, we can't rely on being unmodified

Reply:
I think we can do this as a separate PR tho? what do yall think?

Reply:
I hoped @tuyennhv would do it in separate PR since couple of PR-s depends on this

Reply:
sure